libnmstate/nm/device.py

```diff
@@ -20,13 +20,28 @@
 from . import nmclient
 
 
-def activate(dev, connection=None):
+def activate(dev, fetch_remote_connection_func=None):
+    """
+    Activating the given device or remote connection profile.
```

Comment:
minor detail: Better start the description in imperative mood, i.e. `Activate the given device... `

Reply:
Done